concent_api/concent_api/settings/base.py

```diff
@@ -253,6 +253,10 @@
 #CONCENT_PRIVATE_KEY =
 #CONCENT_PUBLIC_KEY  =
 
+# Private and public keys to be used by Signing Service
+# SIGNING_SERVICE_PUBLIC_KEY =
+# SIGNING_SERVICE_PRIVATE_KEY =
```

Comment:
Should Concent know about SigningService private key ? Issue doesn't mention this.

Reply:
good question, @cameel ?

Reply:
No. The private key is private of course :)

Signing Service is an external service and won't be running on the cluster (except for testing). Concent and Signing Service may only know each other's public keys. Otherwise one could impersonate the other which kinda defeats the purpose of separating them.